explainer/explainer11.md

```diff
@@ -266,7 +294,7 @@ that explain how to encode the metadata needed to interact with different IoT pl
 in a WoT Thing Description.
 
 #### WoT Security and Privacy Considerations
-The final building block is security, which is a cross-cutting concern.
+The final building blocks are security and privacy, which are cross-cutting concerns.
```

Comment:
Please align title and first sentence "Considerations"->Guidelines

Reply:
Actually, I just took "Considerations" out of the title so it's just "WoT Security and Privacy" now.  The "Guidelines" document is the deliverable, but the section discusses a number of other aspects of security and privacy that are addressed in various places so making the title more general makes sense.